hack/print-workspace-status.sh

```diff
@@ -40,4 +40,9 @@ STABLE_gitTreeState ${KUBE_GIT_TREE_STATE-}
 STABLE_gitVersion ${KUBE_GIT_VERSION-}
 STABLE_gitMajor ${KUBE_GIT_MAJOR-}
 STABLE_gitMinor ${KUBE_GIT_MINOR-}
+gitCommit ${KUBE_GIT_COMMIT-}
```

Comment:
Nope. These need to match the variable names here: https://github.com/kubernetes/kubernetes/blob/master/pkg/version/base.go#L42